File: Removing Minimum and Maximum From Array/Removing Minimum and Maximum From Array.java
// Solution for Removing Minimum and Maximum From Array in JAVA

class Solution {
    public int minimumDeletions(int[] nums) {
        int n = nums.length;
        int minIdx = 0, maxIdx = 0;

        for (int i = 1; i < n; i++) {
            if (nums[i] < nums[minIdx]) minIdx = i;
            if (nums[i] > nums[maxIdx]) maxIdx = i;
        }

        int l = Math.min(minIdx, maxIdx);
        int r = Math.max(minIdx, maxIdx);

        return Math.min(r + 1,
                Math.min(n - l, l + 1 + n - r));
    }
}

File: Removing Minimum and Maximum From Array/Removing Minimum and Maximum From Array.py
# Solution for Removing Minimum and Maximum From Array in PY

class Solution:
    def minimumDeletions(self, nums):
        n = len(nums)
        a = nums.index(min(nums))
        b = nums.index(max(nums))

        l, r = min(a, b), max(a, b)

        return min(r + 1, n - l, l + 1 + n - r)
